src/openforms/registrations/contrib/json_dump/tests/test_backend.py

```diff
@@ -0,0 +1,238 @@
+from base64 import b64decode
+from pathlib import Path
+
+from django.core.exceptions import SuspiciousOperation
+from django.test import TestCase
+
+from glom import glom
+from requests import RequestException
+from zgw_consumers.test.factories import ServiceFactory
+
+from openforms.submissions.public_references import set_submission_reference
+from openforms.submissions.tests.factories import (
+    SubmissionFactory,
+    SubmissionFileAttachmentFactory,
+)
+from openforms.utils.tests.vcr import OFVCRMixin
+
+from ..plugin import JSONDumpRegistration
+
+VCR_TEST_FILES = Path(__file__).parent / "files"
+
+
+class JSONDumpBackendTests(OFVCRMixin, TestCase):
```

Comment:
Can you add a test for a `nested/path` endpoint? You'll have to update the Flask app for that too of course.